File: test/benchmarks/lua/fib.lua
local function fib(n)
  if n < 2 then
    return n
  else
    return fib(n - 1) + fib(n - 2)
  end
end

local start = os.clock()

local sum = 0

for i = 1, 10 do
  sum = sum + fib(30)
end

local seconds = os.clock() - start
print('elapsed: ' .. seconds .. 's | ' .. sum)

File: test/benchmarks/python/fib.py
from time import process_time as clock

def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)

def main():
    start = clock()

    result = 0

    for i in range(0, 10):
        result += fib(30)

    seconds = clock() - start
    print("elapsed: " + str(seconds) + "s | " + str(result))

main()
